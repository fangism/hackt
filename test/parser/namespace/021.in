// this namespace test should fail

namespace W {
	namespace X {
		namespace B { }
	}
	namespace Y { }
	namespace Z {
		namespace A {
			open B;		// can't find B, X is not open
		}
	}
}

