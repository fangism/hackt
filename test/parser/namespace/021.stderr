 ... not found, ERROR! [10:9]
